### PR TITLE
Add explaination on vue3 sample

### PR DESCRIPTION
**Pull request by bidoubiwa**
Add missing explaination on vue3 code samples and remove the notification that a vue3 example is on its way

### CODE REVIEW

File: .code-samples-vue-3.meilisearch.yaml [hunk 1]
```diff
@@ -39,9 +39,8 @@ landing_getting_started: |-
   };
   </script>
 
-
 getting_started_front_end_integration_md: |-
-  The following example uses [Vue 3](https://vuejs.org/).
+  The following example uses [Vue 3](https://vuejs.org/) a JavaScript framework for building web user interfaces.
```
Comment (dichotommy) [inline .code-samples-vue-3.meilisearch.yaml:43]:
```suggestion
  The following example uses [Vue 3](https://vuejs.org/), a JavaScript framework for building web user interfaces.
```

File: .code-samples-vue-3.meilisearch.yaml [hunk 2]
```diff
@@ -100,3 +99,9 @@ getting_started_front_end_integration_md: |-
 
   </html>
   ```
+
+  Here's what's happening:
+  - To use `instant-meilisearch` with Vue, you must add `<ais-instant-search>`, `<ais-search-box>`, and `<ais-hits>` to your application's HTML. These components are mandatory when generating the`instant-meilisearch` interface
```
Comment (dichotommy) [inline .code-samples-vue-3.meilisearch.yaml:104]:
```suggestion
  - To use `instant-meilisearch` with Vue, you must add `<ais-instant-search>`, `<ais-search-box>`, and `<ais-hits>` to your application's HTML. These components are mandatory when generating the `instant-meilisearch` interface
```

File: .code-samples-vue-3.meilisearch.yaml [hunk 3]
```diff
@@ -100,3 +99,9 @@ getting_started_front_end_integration_md: |-
...
+  - Other Vue components such as `<ais-configure>` and `<ais-highlight>` are optional. They offer greater control over `instant-meilisearch`'s behavior and appearance
+  - The three two`<script src="..">` tags import libraries needed to run `instant-meilisearch` with Vue
```
Comment (dichotommy) [inline .code-samples-vue-3.meilisearch.yaml:106]:
```suggestion
  - The three `<script src="..">` tags import libraries needed to run `instant-meilisearch` with Vue
```

**Review by dichotommy — CHANGES_REQUESTED** [review]

**Review by dichotommy — APPROVED** [review]
🏄🏻

**Comment by dichotommy** [comment]
I do not have write access, so I am requesting a review from Bruno

**Review by CaroFG — APPROVED** [review]

**Pull request merged**